assembly/std/index.ts

```diff
@@ -140,28 +236,42 @@ export function generateEvent(event: string): void {
 /**
  * Transfers SCE coins from the current address to given address.
  *
- * @param to -
+ * @param to - the address to send coins to.
  * @param amount - value in the smallest unit.
```

Comment:
We can explain what is the smallest unit in a remark.

see: [unit standard](https://github.com/massalabs/massa-standards/blob/main/units.md)
Or just a link. What do you think @gregLibert ?

Reply:
Yes, you can reference the standard in a remark!